activerecord/test/cases/encryption/extended_deterministic_queries_test.rb

```diff
@@ -12,7 +12,7 @@ class ActiveRecord::Encryption::ExtendedDeterministicQueriesTest < ActiveRecord:
     ActiveRecord::Encryption.without_encryption { Book.create! name: "Dune" }
     puts EncryptedBook.where(name: "Dune").to_sql
     assert EncryptedBook.find_by(name: "Dune") # core
-    # assert EncryptedBook.where("id > 0").find_by(name: "Dune") # relation
```

Comment:
sneaking this in as I had remove this assertion by mistake

Reply:
This is already fixed in 27b0180232486f582f08fc3906451ca54b3a127b.

Reply:
oh thanks @kamipo!